src/rdbms/mongoose_rdbms_odbc.erl

```diff
@@ -75,7 +75,16 @@ query(Connection, Query, Timeout) ->
 -spec prepare(Connection :: term(), Name :: atom(), Table :: binary(),
               Fields :: [binary()], Statement :: iodata()) ->
                      {ok, {binary(), [fun((term()) -> tuple())]}}.
-prepare(Connection, _Name, Table, Fields, Statement) ->
+prepare(Connection, Name, Table, Fields, Statement) ->
+    try prepare2(Connection, Table, Fields, Statement)
```

Comment:
I am not convinced to add the extra code only to display the keys if someone makes a mistake in the prepared query...